packages/icons/resources/icons/icons.json

```diff
@@ -3400,5 +3400,11 @@
     "tags": "science, beaker, chemistry, mixing, scientist",
     "group": "interface",
     "content": "\\e90e"
+  },
+  {
+    "displayName": "Virus",
+    "iconName": "virus",
+    "tags": "virus, doctor, medical, examination, disease, pandemic, sick",
+    "group": "interface"
   }
```

Comment:
I removed the "content" field here because it was unused, and I changed the build scripts to allow for this omittance